http_server/_server_connection.pony

```diff
@@ -1,9 +1,10 @@
-use "net"
 use "collections"
-use "valbytes"
 use "debug"
+use "net"
 use "time"
 
+use "valbytes"
```

Comment:
This is unrelated at this point, no new uses were added so I think this should be rolled bacl